configuration/sms_pipeline_config.json

```diff
@@ -215,24 +215,52 @@
     "GenerateMogadishuThemeDistributionMaps": true,
 
     "TrafficLabels": [
-      {"StartDate": "2020-11-01T16:30+03:00", "EndDate": "2020-11-01T17:30+03:00", "Label": "E01 Ad"},
+      {"StartDate": "2020-11-01T16:30+03:00", "EndDate": "2020-11-01T18:30+03:00", "Label": "E01 Ad"},
       {"StartDate": "2020-11-05T14:30+03:00", "EndDate": "2020-11-05T15:30+03:00", "Label": "E01 Dalsan/Danan"},
       {"StartDate": "2020-11-05T15:00+03:00", "EndDate": "2020-11-05T16:00+03:00", "Label": "E01 Risala"},
       {"StartDate": "2020-11-05T11:30+03:00", "EndDate": "2020-11-05T12:30+03:00", "Label": "E01 Star"},
       {"StartDate": "2020-11-05T16:30+03:00", "EndDate": "2020-11-05T17:30+03:00", "Label": "E01 Mustaqbal"},
 
-      {"StartDate": "2020-11-08T16:30+03:00", "EndDate": "2020-11-08T17:30+03:00", "Label": "E02 Ad"},
+      {"StartDate": "2020-11-08T16:30+03:00", "EndDate": "2020-11-08T18:30+03:00", "Label": "E02 Ad"},
       {"StartDate": "2020-11-12T14:30+03:00", "EndDate": "2020-11-12T15:30+03:00", "Label": "E02 Dalsan/Danan"},
       {"StartDate": "2020-11-12T15:00+03:00", "EndDate": "2020-11-12T16:00+03:00", "Label": "E02 Risala"},
       {"StartDate": "2020-11-12T11:30+03:00", "EndDate": "2020-11-12T12:30+03:00", "Label": "E02 Star"},
       {"StartDate": "2020-11-12T16:30+03:00", "EndDate": "2020-11-12T17:30+03:00", "Label": "E02 Mustaqbal"},
 
-      {"StartDate": "2020-11-15T16:30+03:00", "EndDate": "2020-11-15T17:30+03:00", "Label": "E03 Ad"},
+      {"StartDate": "2020-11-15T16:30+03:00", "EndDate": "2020-11-15T18:30+03:00", "Label": "E03 Ad"},
       {"StartDate": "2020-11-18T16:30+03:00", "EndDate": "2020-11-18T17:30+03:00", "Label": "E03 Imaqal Ad"},
       {"StartDate": "2020-11-19T14:30+03:00", "EndDate": "2020-11-19T15:30+03:00", "Label": "E03 Dalsan/Danan"},
       {"StartDate": "2020-11-19T15:00+03:00", "EndDate": "2020-11-19T16:00+03:00", "Label": "E03 Risala"},
       {"StartDate": "2020-11-19T11:30+03:00", "EndDate": "2020-11-19T12:30+03:00", "Label": "E03 Star"},
-      {"StartDate": "2020-11-19T16:30+03:00", "EndDate": "2020-11-19T17:30+03:00", "Label": "E03 Mustaqbal"}
+      {"StartDate": "2020-11-19T16:30+03:00", "EndDate": "2020-11-19T17:30+03:00", "Label": "E03 Mustaqbal"},
+
+      {"StartDate": "2021-02-14T16:30+03:00", "EndDate": "2021-02-14T18:30+03:00", "Label": "E04 IBTCI/FCDO Ad"},
```

Comment:
based on past traffic people respond to the advert even over 24 hours so best to block the time till the first radio show airs for the advert traffic.

Reply:
I expect people do respond after more than a few hours, the problem is they start to compete with the promos e.g. in this traffic graph from the e5 ad, the first peak is the main response to the advert, but the second peak only a few hours later? Most likely a promo, or a mix, it gets very hard to distinguish and report effectively.
<img width="59" alt="image" src="https://user-images.githubusercontent.com/7963608/108833283-80361580-75c4-11eb-903b-1acd266bda7a.png">

Reply:
Proceeding with 2 hrs for now but will flag this to the team for further input once deployed